### PR TITLE
Colorscale bugfixing

### PR DESCRIPTION
**Pull request by dsp0011**

### CODE REVIEW

File: src/demo/example-data/new-layered-map.json [hunk 1]
```diff
@@ -26,6 +26,8 @@
         {
             "name": "A seismic horizon without colormap",
             "id": 2, 
+<<<<<<< HEAD
+=======
```
Comment (eolweus) [inline src/demo/example-data/new-layered-map.json:30]: resolve this (if it's still here)

File: src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ const ColorBar = (props) => {
         if(!focusedImageLayer) {
             return [null, null, null];
         }
+<<<<<<< HEAD
+=======
 
+>>>>>>> interns-dev
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js]:
```suggestion

```

File: src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js [hunk 3]
```diff
@@ -28,7 +28,10 @@ const ColorBar = (props) => {
         if(!focusedImageLayer) {
             return [null, null, null];
         }
+<<<<<<< HEAD
```
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js:31]:
```suggestion
```

File: src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js [hunk 4]
```diff
@@ -28,7 +28,10 @@ const ColorBar = (props) => {
...
+=======
```
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js:32]:
```suggestion
```

File: src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js (hunk 2 above)
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/Colorbar.js:34]:
```suggestion
```

File: src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js [hunk 5]
```diff
@@ -27,27 +27,40 @@ class DrawControls extends Component {
...
     }
 
     componentDidMount() {
         const { drawPolygon, drawMarker, drawPolyline } = this.props;
-        this.addToolbar(this.props.map);
+        this.createDrawControl();
+    }
+
+    componentDidUpdate(prevProps) {
+        if(this.props.drawPolygon != prevProps.drawPolygon
```
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js:42]:
```suggestion
        if(this.props.drawPolygon !== prevProps.drawPolygon
```

File: src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js [hunk 6]
```diff
@@ -27,27 +27,40 @@ class DrawControls extends Component {
...
+           || this.props.drawPolyline != prevProps.drawPolyline
```
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js:43]:
```suggestion
           || this.props.drawPolyline !== prevProps.drawPolyline
```

File: src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js [hunk 7]
```diff
@@ -27,27 +27,40 @@ class DrawControls extends Component {
...
+           || this.props.drawMarker != prevProps.drawMarker
```
Comment (Andorr) [inline src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js:44]:
```suggestion
           || this.props.drawMarker !== prevProps.drawMarker
```

File: src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js [hunk 8]
```diff
@@ -114,6 +114,7 @@ L.ImageWebGLOverlay = L.Layer.extend({
     		options.colorScale = Object.assign(this.options.colorScale, options.colorScale)
    	    }       	   
 
+        // console.log("new colorscale: ", options.colorScale)
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js:117]:
```suggestion
```

**Pull request merged**